mbedtls/Cargo.toml

```diff
@@ -33,7 +33,7 @@ rc2 = { version = "0.3", optional = true }
 rs-libc = "0.1.0"
 
 [dependencies.mbedtls-sys-auto]
-version = "2.18.0"
```

Comment:
Unnecessary bump

Reply:
But the change we want to pick up is in mbedtls-sys-auto isn't it?

Reply:
Or do you mean that it goes from `2.18.0` to `2.18.2`?

Reply:
If 2.18.0 is working fine for people then there's no reason to force them to upgrade.

Reply:
So this means this file doesn't need to change at all.

Reply:
Understood